sdk/armada-protocol-common/src/deployments/index.ts

```
@@ -114,3 +114,36 @@ export const getDeployedGovRewardsManagerAddress = () => {
   }
   return Address.createFromEthereum({ value: maybeAddress })
 }
+
+export const getBridgeContractAddress = (chainInfo: ChainInfo) => {
+  const config = getConfig()
+  const key = getConfigKey(chainInfo.name)
+  return Address.createFromEthereum({
+    value: config[key].deployedContracts.gov.summerToken.address,
```

please add try catch / throw when address undefined